src/jit/alloc.cpp

```diff
@@ -326,10 +326,13 @@ void ArenaAllocator::freeHostMemory(void* block)
 //    version does not: it may inject faults into the allocator and
 //    seeds all allocations with a specified pattern to help catch
 //    use-before-init problems.
+//    The non-DEBUG version is defined in our header file "alloc.h"
```

Comment:
This comment is redundant: the location of the release version is mentioned above.